pages/coinbase/homepage.py

```diff
@@ -0,0 +1,30 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support import expected_conditions as EC 
+from selenium.webdriver.support.ui import WebDriverWait as wait
+from selenium.webdriver.common.by import By
+import secure.cookies
+import config
+
+
+
+class Home_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.coinbase_homepage_url)
+
+    #Buttons
+
+    # coinbase has two-factor authentication which involves sending a text
+    # via cellphone; it is optional to forego this authentication, so 
+    # cookies are added below to do this. It is currently unclear which cookie controls 
+    # the authentication. So, I just signed in, checked the option to not ask for
+    # the text code again, then signed out and grabbed all the cookies. Below I loaded 
+    # all these cookies which include the cookie/s necessary to forego this second 
+    # authentication.
+
+    def home_page_sign_in_button_loc(self): return (By.XPATH,"//a[@title='Sign in']")
+    def click_home_page_sign_in_button(self):
+        self.driver.find_element(*self.home_page_sign_in_button_loc()).click()
+        for cookie in secure.cookies.coinbase: self.driver.add_cookie(cookie)
```

Comment:
Remove this line and the accompanying comment above. The cookie injection should be a method in the /signin object, and it should either be an optional parameter of clicking the sign in button on that page, an optional parameter of a sign in workflow, or both. (When you move this, make sure the accompanying comment goes above the this line, not above the function it's in.)

You don't want it in this method, and you definitely don't want it to be mandatory, because it blocks us from testing two-factor auth.